velox/dwio/parquet/writer/arrow/tests/StatisticsTest.cpp

```diff
@@ -1272,24 +1301,32 @@ TEST_F(TestStatisticsSortOrderFLBA, UnknownSortOrder) {
       "Column 0",
       Repetition::REQUIRED,
       Type::FIXED_LEN_BYTE_ARRAY,
-      ConvertedType::INTERVAL,
-      FLBA_LENGTH));
+      ConvertedType::DECIMAL,
+      FLBA_LENGTH,
+      10,
+      2));
   this->SetUpSchema();
   this->WriteParquet();
 
   ASSERT_OK_AND_ASSIGN(auto pbuffer, parquet_sink_->Finish());
 
-  // Create a ParquetReader instance
-  std::unique_ptr<ParquetFileReader> parquet_reader = ParquetFileReader::Open(
-      std::make_shared<::arrow::io::BufferReader>(pbuffer));
-  // Get the File MetaData
-  std::shared_ptr<FileMetaData> file_metadata = parquet_reader->metadata();
-  std::shared_ptr<RowGroupMetaData> rg_metadata = file_metadata->RowGroup(0);
-  std::shared_ptr<ColumnChunkMetaData> cc_metadata =
-      rg_metadata->ColumnChunk(0);
-
-  // stats should not be set for UNKNOWN sort order
```

Comment:
I just realized that this test is for UNKNOWN sort order.  We have to figure this out.

Reply:
Velox reader does not support any logical types with undefined sort order yet.
https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L1048
Let's fix the test name as DecimalSortOrder and keep this change. Please verify that no such test exists for Decimal already.